html/includes/authentication/ldap.inc.php

```diff
@@ -100,7 +100,7 @@ function get_userlevel($username)
 {
     global $config, $ldap_connection;
 
-    $userlevel = 0;
+    $userlevel = -1;
```

Comment:
I see that the other authentication methods has `0` as their value. For consistency, I'll change it back to `0`.